### PR TITLE
doc: deploy cpi with k3s

### PR DESCRIPTION
**Pull request by nicolehanjing**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
doc: deploy cpi with k3s
```

### CODE REVIEW

File: docs/book/tutorials/deploying-cpi-with-k3s.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# Deploying the vSphere CPI using k3s
+
+This document is designed to show you how to integrate k3s with cloud provider vsphere, which means that your k3s cluster will be able to talk to cloud provider API in order to make requests and configurations. In this tutorial, we will be running k3s with cloud provider vsphere.
+
+When running with a cloud-controller-manager, it is expected to pass the node provider ID to a CCM as `<provider>://<id>`, in our case, `vsphere://1234567`. However, k3s passes it as `k3s://<hostname>`, which makes vsphere CCM reject it.
```
Comment (lubronzhan) [inline docs/book/tutorials/deploying-cpi-with-k3s.md:5]: maybe rephrase to `vsphere CCM won't be able to find the node ` instead of `which makes vsphere CCM reject it.`

**Review by lubronzhan — APPROVED** [review]
lgtm

File: docs/book/tutorials/deploying-cpi-with-k3s.md (hunk 1 above)
Comment (nicolehanjing) [reply docs/book/tutorials/deploying-cpi-with-k3s.md:5]: gotcha, updated!
